### DISCUSSION TITLE
Under what circumstances will this happen?

### DISCUSSION THREAD

**Discussion by Maskvvv** [apache/flink-cdc#2241] [Q&A]
![](http://qiniu.zhouhongyin.top/2023/06/27/1687852015-20230627154654.png)
CREATE -> insert
UPDATE > update
DELETE -> delete
READ -> ???
TRUNCATE -> ???

**Comment by leonardBang** [comment]
IIUC, READ means 'inert' record from history records of a table,  TRUNCATE means 'delete' record that produced by TRUNCATE command.

**Reply by Maskvvv** [reply]
Thank you for your reply. I'd like to ask you two more questions.
1. How to understand the sentence "'insert' record from history records of a table"? 
2 What's the difference between READ and CRATE?
